src/pages/user_profile/RequestRole.js

```diff
@@ -0,0 +1,187 @@
+import { useState } from 'react';
+import { div, h, h2, p, button } from 'react-hyperscript-helpers';
```

Comment:
Not required for this PR, but one thing we've been doing is trying to remove usages of `react-hyperscript-helpers` when we rewrite components. It's a relatively straight-forward switch ... just use standard jsx `<button>` instead of hyperscript's `button({})`

Reply:
Sure! I can start doing that from the next component onwards. I'll also add a comment to the "Finalizing components" ticket to make a change from hyperscript to jsx in all files related to user profile.